### PR TITLE
Fixed creating user

### LINKED ISSUE #23327: User cannot be created via Admin UI if custom user attribute is required
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

admin/ui

### Describe the bug

When having the user-profile feature enabled and having a custom user attribute marked as required, admin cannot create user via the Admin UI because the attribute is not displayed in the user creation screen, therefore cannot be provided. The field is only shown when editing user and can be set there (have to make the attribute optional)...

### Version

22.0.3

### Expected behavior

Admin is able to create user via Admin UI when having custom user attributes marked as required.

### Actual behavior

Admin is not able to create user via Admin UI when having custom user attributes marked as required.

### How to Reproduce?

1. Enable the user-profile and add new custom user attribute (for e.g locale) and mark it as required
2. Try to create user as an admin

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by agagancarczyk**
Closes https://github.com/keycloak/keycloak/issues/23327

### CODE REVIEW

File: js/apps/admin-ui/src/user/CreateUser.tsx [hunk 1]
```diff
@@ -19,6 +19,8 @@ import { UserFormFields, toUserRepresentation } from "./form-state";
 import { toUser } from "./routes/User";
 
 import "./user-section.css";
+import RealmRepresentation from "libs/keycloak-admin-client/lib/defs/realmRepresentation";
```
Comment (jonkoops) [inline js/apps/admin-ui/src/user/CreateUser.tsx:22]: This import should be coming from `@keycloak/keycloak-admin-client`

File: js/apps/admin-ui/src/user/EditUser.tsx [hunk 2]
```diff
@@ -124,7 +124,11 @@ export default function EditUser() {
     try {
       await adminClient.users.update(
         { id: user!.id! },
-        toUserRepresentation(data),
+        {
```
Comment (jonkoops) [inline js/apps/admin-ui/src/user/EditUser.tsx:127]: The `toUserRepresentation()` function still needs to be called on the data being passed here.

**Review by jonkoops — CHANGES_REQUESTED** [review]
Looks like this PR fixes several bugs, if they are two distinct bugs I would prefer we have two PRs instead. The reason for this is that we can only link a single issue per commit, otherwise one of them will not show up in the release notes.

**Review by ssilvert — APPROVED** [review]
LGTM.  I'd also like @edewit to take a look and approve before merging.

**Review by jonkoops — APPROVED** [review]
LGTM!

**Pull request merged**